legend-engine-xts-openapi/legend-engine-xt-openapi-generation/src/test/java/org/finos/legend/engine/generation/TestOpenApiArtifactGenerationExtension.java

```diff
@@ -14,6 +14,7 @@
 
 package org.finos.legend.engine.generation;
 
+import org.finos.legend.engine.generation.extension.OpenApiArtifactGenerationExtension;
```

Comment:
??

Reply:
yes, an instance of OpenApiArtifactGenerationExtension is being created in the test